enhancements/sdk-fbc-run-bundle.md

```diff
@@ -0,0 +1,279 @@
+---
+title: sdk-fbc-run-bundle
+authors:
+  - "@rashmigottipati"
+reviewers:
+  - "@joelanford"
+  - "@jmrodri"
+  - "@VenkatRamaraju"
+approvers:
+  - "@joelanford"
+  - "@jmrodri"
+  - "@VenkatRamaraju"
+creation-date: 2022-04-20
+last-updated: 2022-04-20
+status: implementable
+see-also:
+replaces:
+superseded-by:
+
+---
+
+# Add FBC support to run bundle
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [X] Design details are appropriately documented from clear requirements
+- [X] Test plan is defined
+- [X] Graduation criteria for dev preview, tech preview, GA
+
+## Open Questions
+
+- Do we need to build a File-Based Catalog to run this command?
+
+  - In the case where an index image isn't provided, we need to build a File-Based Catalog on the fly when `run bundle` command is invoked.
+
+- Do we need to make our own image to use as a default one for `run bundle` and `run bundle-upgrade`?
+
+  - Currently, we are using `quay.io/operator-framework/opm:latest` as the default index image. This image should suffice as the default image for both index formats (FBC and SQLite images).
+
+- While initializing the FBC, should the channel received from the bundle annotations be used to initialize the default channel name in the `opm init` command?
+
+  - We should be able to handle bundles that don't have channel metadata on the annotations.
```

Comment:
The answer here says that we should be able to handle bundles without this info but does not answer the question. Should we use or not the annotations when the data is informed?

Reply:
@camilamacedo86 I think bundles won't have to have channel metadata included going forward. So we should be able to handle channel assignment when the channel metadata doesn't exist. 

When a bundle contains the annotation, we use it as the channel name to create the FBC. If doesn't have one, then we generate a random name or a non-random name with a meaningful prefix/suffix. Going by the discussions, it seems that a randomly generated name would be better. 

I've added this detail in the EP as well for this question.